composer.json

```diff
@@ -37,12 +37,11 @@
         "source": "https://github.com/MyIntervals/emogrifier"
     },
     "require": {
-        "php": "^5.5.0 || ~7.0.0 || ~7.1.0 || ~7.2.0",
+        "php": "^5.5.0 || ~7.0.0 || ~7.1.0 || ~7.2.0 || ~7.3.0",
```

Comment:
Could you please undo the change? I'd like to keep a "only one issue per PR" policy so that git blame is more meaningful, and so that we can revert individual changes more easily.